operator/README.md

```diff
@@ -1,134 +1,17 @@
-1. Create an Infinispan cluster with three nodes.
-```bash
-$ oc apply -f https://raw.githubusercontent.com/infinispan/infinispan-operator/0.3.0/deploy/cr/cr_minimal.yaml
-```
+Complete this tutorial to learn how to use the Infinispan Operator for
+Kubernetes clusters.
 
-2. Verify that the Infinispan cluster forms.
-```
-$ oc get pods -l app=infinispan-pod
-NAME                                   READY     STATUS
-example-infinispan-<id_1>     1/1       Running
-example-infinispan-<id_2>     1/1       Running
-example-infinispan-<id_3>     1/1       Running
-```
-Where `<id_*>` is the generated identifier for the pod.
-```
-$ oc logs example-infinispan-b58cb5699-727zq
-...
-INFO  [org.infinispan.CLUSTER] (MSC service thread 1-2)
-ISPN000094: Received new cluster view for channel cluster:
-[example-infinispan-<id_1>|2] (3)
-[example-infinispan-<id_1>,example-infinispan-<id_2>,
-example-infinispan-<id_3>]
-```
+Get the docs for this tutorial, either:
 
-Connecting to Infinispan Clusters
----------------------------------
-1. Expose the Infinispan service for external access.
-```
-$ oc expose svc example-infinispan
-```
+* Visit the [Infinispan Tutorials](https://infinispan.org/tutorials/).
+* Clone this repository and then build the documentation locally, as follows:
 
-2. Export the host for the public route to a local variable.
-```
-$ export INFINISPAN_HOST=$(oc get route example-infinispan -o jsonpath="{.spec.host}")
-```
-
-Storing and Retrieving Data
----------------------------
-Infinispan requires authentication for data access. The default user is `developer` and the Operator automatically generates a password and stores it in a secret when the cluster starts.
-
-1. Export the generated password from the secret to a local variable.
-```
-$ export PASS=$(oc get secret example-infinispan-app-generated-secret -o jsonpath="{.data.password}" | base64 --decode)
-```
-
-2. Store some data through the HTTP endpoint.
-```
-$ curl -v \
-    -X POST \
-    -u developer:${PASS} \
-    -H 'Content-type: text/plain' \
-    -d 'test-value' \
-    ${INFINISPAN_HOST}/rest/default/test-key
-...
-< HTTP/1.1 200 OK
-```
-
-2. Retrieve the data from the Infinispan cluster.
-```
-$ curl -v \
-    -u developer:${PASS} \
-    ${INFINISPAN_HOST}/rest/default/test-key
-...
-< HTTP/1.1 200 OK
-...
-test-value
-```
-
-  You've successfully completed this tutorial!
+  ```
+  $ git clone git@github.com:infinispan/infinispan-simple-tutorials.git
 
-  Use the following commands to free up the resources that you created:
+  $ cd infinispan-simple-tutorials
 
-  ```bash
-  $ oc delete route example-infinispan
-  $ oc delete infinispan example-infinispan
-  $ oc delete deployment infinispan-operator
-  $ oc delete role infinispan-operator
-  $ oc delete rolebinding infinispan-operator
-  $ oc delete serviceaccount infinispan-operator
-  $ oc delete crd infinispans.infinispan.org
+  $ asciidoctor operator/documentation/asciidoc/operator.asciidoc
```

Comment:
Hmmm, users wanting to try out the tutorial shouldn't have to build the documentation of the tutorial.

We should point them to a URL where they can browse through the tutorial documentation. It's fine if we don't yet have the URL, but should be produced fairly soon after this PR is integrated.

We'd also want to have a way to rebuild the tutorial documentation automatically when a commit that changes the documentation is received.